dhis-2/dhis-support/dhis-support-system/src/test/java/org/hisp/dhis/system/util/ValidationUtilsTest.java

```diff
@@ -620,4 +624,33 @@ void testIsValidLetter()
         List<String> invalid = List.of( "1", "", "aa", "=", "," );
         invalid.forEach( value -> assertFalse( isValidLetter( value ) ) );
     }
+
+    @Test
+    void shouldPassValidationWhenRenderTypeIsCanvasAndValueTypeIsImageAndNoOptionSet()
+    {
+        // Both ProgramStageDataElement and ProgramTrackedEntityAttribute should be supported.
+        assertTrue( validateRenderingType( ProgramStageDataElement.class, IMAGE, false, ValueTypeRenderingType.CANVAS ),
```

Comment:
I don't see the point of commenting inside methods. If it means something, it should be in the test name